### ISSUE TITLE
Support ES6 default exports in cosmos/webpack configs

### ISSUE THREAD

**Issue by skidding** [react-cosmos/react-cosmos#242]
Right now both `cosmos.config.js` and `webpack.config.js` need to export their contents via `module.exports = {` instead of `export default {`. This happens because they are imported via dynamic `require` calls instead of ES6 import (which would *just work* due to Babel's interop boilerplate).

This has already been done for fixtures [here](https://github.com/react-cosmos/react-cosmos/blob/dbfc08ca60ed2ee195c736fa3140907cf9903b31/packages/react-cosmos/src/load-modules.js#L4-L14). The `importModule` helper could be moved in the [react-cosmos-utils package](https://github.com/react-cosmos/react-cosmos/tree/dbfc08ca60ed2ee195c736fa3140907cf9903b31/packages/react-cosmos-utils). The same helper must be applied in these places:
- [Where user webpack.config is imported](https://github.com/react-cosmos/react-cosmos/blob/dbfc08ca60ed2ee195c736fa3140907cf9903b31/packages/react-cosmos-webpack/src/server.js#L25)
- [Where cosmos.config is imported 1](https://github.com/react-cosmos/react-cosmos/blob/dbfc08ca60ed2ee195c736fa3140907cf9903b31/packages/react-cosmos-webpack/src/server.js#L15)
- [Where cosmos.config is imported 2](https://github.com/react-cosmos/react-cosmos/blob/dbfc08ca60ed2ee195c736fa3140907cf9903b31/packages/react-cosmos-webpack/src/webpack-config.js#L16)

**Comment by skidding** [comment]
Done in #244 🙇

**Comment by skidding** [comment]
Published under [`2.0.0-beta.3`](https://github.com/react-cosmos/react-cosmos/tree/v2.0.0-beta.3)

**Comment by Planckit** [comment]
Using es6 import export in webpack.config.js or webpack.config.babel.js isn`t working.

react-cosmos@4.5.0 
webpack@4.15.1

Get **SyntaxError: Unexpected token export**
with es6 export in cosmos.config.js
```
export default {
```

or without es6 in config **SyntaxError: Unexpected token import** in webpack.config.babel.js

**Comment by skidding** [comment]
Hi @Planckit,

Cosmos changed a bit since 2016 and this issue is misleading because reverted what was implemented in this issue down the road.

The config files are from by Node natively. So if your local Node version supports ES modules it will work, otherwise you'll get the error you referenced.

If you want to use Babel features in your configs you can run Cosmso with `babel-node`. We do this in the monorepo https://github.com/react-cosmos/react-cosmos/blob/d9750e777eb9dfe8e1bc5f8c54fda70134b4deb9/package.json#L21

**Comment by Planckit** [comment]
Please Reopen

**Comment by Planckit** [comment]
Tested under 
Node@10.7.0 <- latest at 24.07.18

```
D:\Dev\PrivateProjects\workflow\config\webpack\webpack.config.babel.js:1
(function (exports, require, module, __filename, __dirname) { import general from './webpack.general.babel.js';
                                                                     ^^^^^^^


SyntaxError: Unexpected identifier
```

Some suggestions on how to use webpack.config.js (webpack.config.babel.js) with es6 import?

**Comment by skidding** [comment]
> Some suggestions on how to use webpack.config.js (webpack.config.babel.js) with es6 import?

Yes, I just said "If you want to use Babel features in your configs you can run Cosmos with babel-node. We do this in the monorepo"

So in your package.json scripts, provided you have [babel-node](https://babeljs.io/docs/en/babel-cli#babel-node) installed in your project, you can change this:

```diff
-  "cosmos": "cosmos"
+  "cosmos": "babel-node cosmos"
```

> Node@10.7.0

ES Modules might be available in `10.7`, but they're experimental and need to be [enabled via a flag](https://nodejs.org/api/esm.html#esm_enabling).
